packages/affiliates/apps/OpenDappMiningIssue.js

```diff
@@ -0,0 +1,20 @@
+require("dotenv").config();
+const assert = require("assert");
+const { makeUnixPipe, dappMiningTemplate, createGithubIssue } = require("../libs/affiliates/utils");
+
+const App = env => async params => {
+  const { config } = params;
+  assert(config, "requires config");
+  const issueTemplate = await dappMiningTemplate(config);
+  const githubIssue = await createGithubIssue({ auth: env.github, ...issueTemplate });
+  return {
+    // data.number is the issue number
+    issueNumber: githubIssue.data.number,
+    issueTemplate,
+    ...params
```

Comment:
this opens an issue, and also logs out the issue number so it can be consumed by another app. this would typically be needed when creating the PR to reference the issue number.